code/datums/station_traits/postive_traits.dm

```diff
@@ -0,0 +1,220 @@
+/datum/station_trait/galactic_grant
+	name = "Galactic grant"
+	trait_type = STATION_TRAIT_POSITIVE
+	weight = 5
+	show_in_report = TRUE
+	report_message = "Your station has been selected for a special grant. Some extra funds has been made available to your cargo department."
+
+/datum/station_trait/galactic_grant/on_round_start()
+	GLOB.station_money_database.credit_account(GLOB.station_money_database.get_account_by_department(DEPARTMENT_SUPPLY), rand(2000, 4000), "Galactic Grant", "Great Galactic Grant Group", supress_log = FALSE)
+
```

Comment:
```suggestion
/datum/station_trait/galactic_grant/on_round_start()
	GLOB.station_money_database.credit_account(SSeconomy.cargo_account, rand(2000, 4000), "Galactic Grant", "Great Galactic Grant Group")
```
SSeconomy tracks the supply department account for ease of access, also removed supress_log param since the default is already false